src/less/field/stories/helper-texts-stacked.stories.js

```diff
@@ -32,6 +32,24 @@ export const descriptionCharacterMeter = () => `
 </span>
 `;
 
+export const longDescriptionCharacterMeter = () => `
+<span class="field" style="width:200px">
```

Comment:
I think this should also have the `fluid` class, since this specific example _only_ works when the field is exactly 200px wide and will break if design ever has us change the default width of textboxes

Reply:
I think its fine as is. 
This is mostly to demonstrate that we need a fixed width or fluid to wrap (as in other cases)